extensions/mssql/src/features.ts

```diff
@@ -27,6 +27,41 @@ export class TelemetryFeature implements StaticFeature {
 	}
 }
 
+export class AccountFeature implements StaticFeature {
+
+	constructor(private _client: SqlOpsDataClient) { }
+
+	fillClientCapabilities(capabilities: ClientCapabilities): void {
+		Utils.ensure(capabilities, 'telemetry')!.telemetry = true;
+	}
+
+	initialize(): void {
+		this._client.onRequest(contracts.SecurityTokenRequest.type, async e => {
+			const accountList = await azdata.accounts.getAllAccounts();
+
+			if (accountList.length < 1) {
+				// TODO: Prompt user to add account
+				window.showErrorMessage('Azure Data Studio needs to contact Azure Key Vault to access a column master key for Always Encrypted, but no linked Azure account is available. Please add a linked Azure account and retry the query.');
```

Comment:
These messages should be localized

Reply:
Example https://github.com/microsoft/azuredatastudio/blob/4a68ab46590134be0fa663cb59ac59b8603df1d1/extensions/admin-tool-ext-win/src/main.ts#L114

Reply:
Fixed.